text/src/autogluon/text/automm/data/dataset.py

```diff
@@ -69,8 +75,16 @@ def __getitem__(self, idx):
         Input data formatted as a dictionary.
         """
         ret = dict()
-        for per_modality, per_modality_processors in self.processors.items():
-            for per_model_processor in per_modality_processors:
-                ret.update(per_model_processor(getattr(self, per_modality), idx, self.is_training))
-
+        try:  # TODO: consider handling image exceptions within the process_image.py.
+            # TODO: If no images are open, then throw an exception to be captured here.
+            for per_modality, per_modality_processors in self.processors.items():
+                for per_model_processor in per_modality_processors:
+                    ret.update(per_model_processor(getattr(self, per_modality), idx, self.is_training))
+        except Exception as e:
```

Comment:
Would we increase `self._consecutive_errors += 1` ?

Reply:
Right. Added.